front/src/View/Pages/Inscription.tsx

```diff
@@ -0,0 +1,197 @@
+import React, {ChangeEvent, FormEvent, ReactNode} from 'react';
+import Request from '../../API/Request';
+import '../../css/Inscription.css';
+import DropBox from '../General/DropBox';
+
+class Inscription extends React.Component{
```

Comment:
use states

Reply:
but we don't hace state in connection page no ? for the input variable i mean